### ISSUE TITLE
Use Symfony FOSUserBundle

### ISSUE THREAD

**Issue by myspulin** [forkcms/forkcms#1961]
## Type

- Enhancement

## Problem description

Hi, I would like to use some Symfony bundle which has the dependency to FOSUserBundle. As you probably know this bundle manage users and authentication process. It would be nice to have it because it is quite used in many bundles. Is there any plan to use it in the future?

Thanks for your time. 

## Expected behavior / Proposed solutions

Handle Users by Symfony FOSUserBundle

**Comment by StijnVrolijk** [comment]
https://jolicode.github.io/fosuserbundle-conf/#/

Other than that, we will probably be rewriting our own profiles modules instead of just using a 3rd party bundle. At SumoCoders we've used the FOSUserBundle in our Symfony applications for a few years but recently we've switched to our own MultiUserBundle since it better suits our needs. https://packagist.org/packages/sumocoders/framework-multi-user-bundle

It's highly likely we will be integrating this package (or a version of it) in some form in Fork.
